google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -48,23 +48,19 @@ class Bucket
         attr_accessor :gapi
 
         ##
-        # A boolean value or a project ID string for a requester pays
-        # bucket and its files. If this attribute is set to `true`, transit
-        # costs for operations on the bucket will be billed to the current
-        # project for this client. (See {Project#project} for the ID of the
-        # current project.) If this attribute is set to a project ID, and that
-        # project is authorized for the currently authenticated service account,
-        # transit costs will be billed to the that project. The default is
-        # `nil`.
+        # A boolean value or a project ID string to indicate the project to
+        # be billed for operations on the bucket and its files. If this
+        # attribute is set to `true`, transit costs for operations on the bucket
+        # will be billed to the current project for this client. (See
+        # {Project#project} for the ID of the current project.) If this
+        # attribute is set to a project ID, and that project is authorized for
+        # the currently authenticated service account, transit costs will be
+        # billed to the that project. The default is `nil`.
```

Comment:
This is great, what do you think about adding an additional line for?

The parameter `user_project` is required with requester pays enabled bucket.

Reply:
SGTM, I'll add this everywhere these `user_project` docs appear.